### ISSUE TITLE
Ps4 controller

### ISSUE THREAD

**Issue by boymfriemd** [luckydog7/Funkin-android#175]
I discovered that friday night funkin' is playable with controller but with the arrow i want to know if i you could make a version with the triangle and the button thingys :D

**Comment by DanisLixo** [comment]
just do it kid

**Comment by boymfriemd** [comment]
I cant just because it not work, and pressing triangle i die instantly

**Comment by DanisLixo** [comment]
try the d pad

**Comment by DanisLixo** [comment]
I forget the times that nobody played the original version 😐

**Comment by boymfriemd** [comment]
DanisLixo, yeah i know with the d pad but i just wanted to know if luckydog could change it, if he can't well... :(

**Comment by DanisLixo** [comment]
no, is that the buttons on the right side of the control do not work on the Pc too :)

**Comment by DanisLixo** [comment]
that's what I wanted to do

**Comment by boymfriemd** [comment]
ohh... well that's very sad :(

**Comment by kffwlp** [comment]
You can use the D-Pad?

**Comment by Lstrk** [comment]
Try in octopus app
